proto/spire/api/server/localauthority/v1/localauthority.proto

```diff
@@ -83,7 +83,19 @@ service LocalAuthority {
 message GetJWTAuthorityStateRequest {}
 
 message GetJWTAuthorityStateResponse {
-        repeated AuthorityState states = 1;
+        // Authority currently being used for signing operations.
+        AuthorityState active = 1;
+
+        // Authority added on bundle but is not used yet.
+        AuthorityState prepared = 2;
+
+        // Authority in that was previously used for signing operations,
+	// but it is not longer.
+        AuthorityState old = 3;
+
+        // Authority without state, This in an error condition and should 
+	// never be encountered.
+        AuthorityState unknown = 4;
```

Comment:
I don't see a good reason to return this. The caller can't do anything with it. If this situation was ever encountered, server logs would probably suffice?

Reply:
agree, there is no reason to return unknown authorities, even we must never get into this situation